### PR TITLE
DDFFORM-66 - Add fulltext search to `articles` and `events` views

### PR DESCRIPTION
**Pull request by kasperbirch1**
#### Link to issue
https://reload.atlassian.net/browse/DDFFORM-66

#### Description
This pull request implements a full-text search filter for the `articles` and `events` views, allowing users to perform searches across the `title`, `description`, and `teaser` fields. The search API has been enhanced to index these fields, and case sensitivity has been disabled to make the search more user-friendly.

image
image

image
image

#### Test
https://varnish.pr-1541.dpl-cms.dplplat01.dpl.reload.dk/artikler
https://varnish.pr-1541.dpl-cms.dplplat01.dpl.reload.dk/arrangementer

### CODE REVIEW

File: web/themes/custom/novel/templates/views/input--search-api-fulltext.html.twig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{# web/core/themes/stable9/templates/form/input.html.twig #}
+<input{{ attributes.addClass('search-full-text__input') }} />
+<input type="image" src="/themes/custom/novel/assets/dpl-design-system/icons/basic/icon-search.svg" alt="Søgeknap">
```
Comment (rasben) [inline web/themes/custom/novel/templates/views/input--search-api-fulltext.html.twig:3]:
```suggestion
<input type="image" src="/themes/custom/novel/assets/dpl-design-system/icons/basic/icon-search.svg" alt="{{ 'Search button'|trans }}">
```

File: web/themes/custom/novel/templates/views/form--views-exposed-form.html.twig [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{# web/core/themes/stable9/templates/form/form.html.twig #}
```
Comment (rasben) [inline web/themes/custom/novel/templates/views/form--views-exposed-form.html.twig:1]: Why this comment?

**Review by rasben — CHANGES_REQUESTED** [review]
Tiny request

File: web/themes/custom/novel/templates/views/form--views-exposed-form.html.twig (hunk 2 above)
Comment (kasperbirch1) [reply web/themes/custom/novel/templates/views/form--views-exposed-form.html.twig:1]: This is where the file is copied from

File: web/themes/custom/novel/templates/views/input--search-api-fulltext.html.twig (hunk 1 above)
Comment (xendk) [reply web/themes/custom/novel/templates/views/input--search-api-fulltext.html.twig]: Neither "Søgeknap" nor "Search button" is correct. The alt text should be the alternative representation if the image isn't used, so it should be "Search".

**Review by rasben — APPROVED** [review]

**Pull request merged**